libs/challenge-registry/user-profile/src/lib/user-profile-stats/user-profile-stats.component.scss

```diff
@@ -1,172 +1,28 @@
-.basic-stats-container {
-  width: 1037px;
-  height: 100px;
+.stats-group {
   display: flex;
-  position: relative;
-  align-items: flex-start;
-  flex-direction: column;
-}
-.basic-stats-logged-in {
-  left: -1px;
-  width: 1037px;
-  height: 100px;
-  display: flex;
-  position: absolute;
-}
-.basic-stats-frameorganizations {
-  top: 0px;
-  left: 0px;
-  width: 225px;
-  height: 100px;
-  display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-  flex-shrink: 1;
-  border-style: solid;
-  border-width: 1px;
-}
-.basic-stats-text {
-  top: 13px;
-  left: 103px;
-  height: auto;
-  position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.basic-stats-text01 {
-  top: 56px;
-  left: 54px;
-  height: auto;
-  position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+  justify-content: center;
 }
-.basic-stats-framestars {
-  top: 0px;
-  left: 224px;
-  width: 225px;
-  height: 100px;
+.stat-item {
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-  flex-shrink: 1;
+  flex-direction: column;
+  flex-basis: 100%;
+  flex: 1;
+  height: 54px;
+  max-width: 120px;
+  padding: 18px 5px 8px;
   border-style: solid;
   border-width: 1px;
```

Comment:
There are two borders added between the stats box:  

<img width="592" alt="Screen Shot 2022-09-23 at 11 25 29 AM" src="https://user-images.githubusercontent.com/73901500/191996755-7872859c-a553-4796-a237-6206dc351d1a.png">

Based on the [figma](https://www.figma.com/file/yAeIrg6Whc8Bqj2EnG4MWP/Challenge-Registry?node-id=31%3A2863), we only need one border. Can you add `border-right-width: 0px;` in its parent, like?
```scss
.stats-group {
  .stat-item:first-child {
    border-right-width: 0px;
  }
}
```

Sorry I didn't catch that earlier.